### ISSUE TITLE
Applying Track statistics tags

### ISSUE THREAD

**Issue by ndjamena** [mbunkus/mkvtoolnix#1507]
Now that MediaInfo should be getting proper tag handling in the next version...

What's the chance of adding a mode in MKVInfo or MKVPropEdit that adds the statistics tags to files that otherwise have none... just for the benefit of those people who chose to remove them or are using mkvs from a different muxer.

It would be nice when people start complaining on forums to be able to give them a method to add the tags that doesn't involve recreating the entire file.

**Comment by mbunkus** [comment]
Thanks for posting it here, too.

**Comment by ndjamena** [comment]
MakeMKV has begun adding it's own Tags in version 1.9.8.

So far I've found NUMBER_OF_BYTES_UNCOMPRESSED for anything compressed with zlib and SOURCE_ID... which is the PID of the track from the original Blu Ray.  I'm not sure that second one needs to be added to _STATSTICS_TAGS, but... there it is.

I've tried looking at FFMPEG and VLC to see if I can figure out how to get them to process the tags better, but like MKVToolNix itself they refuse the compile on my lowly copy of Visual Studio 2012.

I'm afraid we'll have to wait for someone with more know-how and resources then me to take an interest before anything can be done there.
